cmd/ccboc/cmd/root.go

```diff
@@ -0,0 +1,130 @@
+package cmd
```

Comment:
This file should be the `main.go`

Reply:
This is the hierarchy according to cobra's manual page.

Reply:
For this case I don't think that we need it. You can just move all stuff from `root.go` to `main.go` and move the `commands.go` file in the `package main`

Reply:
Otherwise, you can follow the hierarchy but for all the commands. See for example: https://github.com/kubernetes/kubectl/tree/master/pkg/cmd

Reply:
I think we should keep it like this in case we want to add some new features. This way is more clean imo and sustainable. But let me know and I will do as you say.

Reply:
It is cleaner, but you will have to do it for all the commands.

Reply:
`get`, `create`, `login`

Reply:
The best solution to this might be to rename the file `commands.go` to something else.
https://github.com/spf13/cobra/blob/master/user_guide.md
This says that all the cobra cli commands/flags should be in cmd/root.go